charts/appdaemon/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for Appdaemon.
+
+image:
+  repository: acockburn/appdaemon
+  pullPolicy: IfNotPresent
+  tag: 4.0.5
+
+strategy:
+  type: Recreate
+
+service:
+  port:
+    port: 5050
+  # # Boilerplate config for code-server functionality, be sure to make changes to suit your environment
+  # additionalServices:
+  # - enabled: true
+  #   nameSuffix: vscode
+  #   type: ClusterIP
+  #   port:
+  #     port: 12321
+  #     name: vscode
+  #     protocol: TCP
+  #     targetPort: 12321
+
+env: {}
+  # HA_URL:
+  # DASH_URL:
+  # TOKEN:
+
+# # Boilerplate config for code-server functionality, be sure to make changes to suit your environment
+# additionalContainers:
+# - name: vscode
+#   image: codercom/code-server:3.7.2
+#   imagePullPolicy: IfNotPresent
+#   workingDir: /www/assets
+#   args:
+#   - "--port"
+#   - "12321"
+#   - "--auth"
+#   - "none"
+#   ports:
+#   - name: vscode
+#     containerPort: 12321
+#     protocol: TCP
+#   volumeMounts:
+#   - name: config
+#     mountPath: /www/assets
+#   securityContext:
+#     runAsUser: 0
+
+ingress:
+  enabled: false
+  # # Boilerplate config for code-server functionality, be sure to make changes to suit your environment
+  # additionalIngresses:
+  # - enabled: true
+  #   nameSuffix: "vscode"
+  #   serviceName: appdaemon-vscode
+  #   servicePort: 12321
+  #   annotations:
+  #     kubernetes.io/ingress.class: "nginx"
+  #   hosts:
+  #   - host: appdaemon-editor.k8s-at-home.com
+  #     paths:
+  #     - path: /
+  #       pathType: Prefix
+  #   tls:
+  #   - hosts:
+  #     - appdaemon-editor.k8s-at-home.com
+
+persistence:
+  config:
+    enabled: false
+    emptyDir: false
+    mountPath: /conf
+    ## Persistent Volume Storage Class
+    ## If defined, storageClassName: <storageClass>
+    ## If set to "-", storageClassName: "", which disables dynamic provisioning
+    ## If undefined (the default) or set to null, no storageClassName spec is
+    ##   set, choosing the default provisioner.  (gp2 on AWS, standard on
+    ##   GKE, AWS & OpenStack)
+    # storageClass: "-"
+    # accessMode: ReadWriteOnce
+    # size: 1Gi
+    ## Do not delete the pvc upon helm uninstall
+    # skipuninstall: false
```

Comment:
This is confusing to me. `skipuninstall: false` says to leave the pvc when uninstalled via helm?

Reply:
This is carried over for all the charts using the common library 😆  It adds a helm hook to the PVC.

```metadata:
  name: {{ $pvcName }}
  {{- if or $values.skipuninstall $values.annotations }}
  annotations:
    {{- if $values.skipuninstall }}
    "helm.sh/resource-policy": keep
    {{- end }}```

So, if true, keep PVC when uninstalling with helm.

Reply:
Right, the comment implies that setting it to false would keep the pvc, but that doesn't seem to match what I would expect, or what is actually happening here.

Reply:
Very true, I think we need to update this across all charts starting with this one. Good catch!